### DISCUSSION TITLE
Community video v2.0

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1534] [General]
What about doing another community video similar to this one? https://twitter.com/eddiejaoude/status/1378668010676838403

I saw people writing good night in different languages, this also works well with the great idea from @Eke#3219 about doing another community video https://twitter.com/evavic44/status/1390451115947352066 

what do you all think?

**Comment by eddiejaoude** [comment]
Who wants to be involved 👇

**Reply by kaiwalyakoparkar** [reply]
Me!!!

**Reply by verma-kunal** [reply]
Yeah! Me too 😃

**Reply by Evavic44** [reply]
Count me in. 😎

**Comment by verma-kunal** [comment]
Rules:
- write something that answers this question "why should you join EddieHub"
- A4 paper, folded in half
- recored in landscape

flow
1. receive the paper from the left
2. open paper and show
3. pass to the right
I think we can skip the writing step

**Reply by starlightknown** [reply]
Reference video [Here ](https://youtu.be/QI6qSrBs6BA)

**Reply by verma-kunal** [reply]
So, we would be doing like this or same as the previous one (by the above rules ) ?

**Comment by eddiejaoude** [comment]
Next video discussion here ... https://github.com/EddieHubCommunity/support/discussions/1883
